packages/electron/src/main/modules/BacktraceMainElectronModule.ts

```diff
@@ -96,14 +97,19 @@ export class BacktraceMainElectronModule implements BacktraceModule {
                 app.setPath('crashDumps', options.database.path);
             }
 
+            const getElectronDefaultAttributes = (attributes: Record<string, AttributeType> = attributeManager.get('scoped').attributes) => {
```

Comment:
instead of supplying a default value here, why not just pass it on line 108?

Reply:
I think its a better idea. I'm chaning my code.